### PR TITLE
Expenses

### PR DESCRIPTION
**Pull request by keishaharris**
added Layout component to the expense feature and added an expense route on the side menu

### CODE REVIEW

File: src/components/layout.jsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default function Layout({ children }) {
                     <ActiveLink pathname={pathname} href="/">Dashboard</ActiveLink>
                     <ActiveLink pathname={pathname} href="/work-systems">Work Systems</ActiveLink>
                     <ActiveLink pathname={pathname} href='/goals'>Goals</ActiveLink>
+                    <ActiveLink pathname={pathname} href="/expense">Expense</ActiveLink>
```
Comment (thelmaboamah) [inline src/components/layout.jsx]: This should be plural, right? Expenses?

File: src/pages/expense.jsx [hunk 2]
```diff
@@ -50,6 +51,13 @@ const expense = () => {
     )
 }
 
+expense.getLayout = function getLayout(page) {
```
Comment (thelmaboamah) [inline src/pages/expense.jsx]: Look like you missed capitalizing this expense component. Also, I'd seen a pattern of including Page in the name for the page-level component in another project and adopted it for GoalsPage and Arden used it for other pages. For consistency, could you renaming the component to ExpensesPage?

File: src/pages/expense.jsx [hunk 3]
```diff
@@ -3,6 +3,7 @@ import { Card, CardBody } from "@chakra-ui/react";
 import PageTitle from "@/components/PageTitle";
 import transactions from "@/lib/transactions";
 import {create} from 'zustand';
+import Layout from "@/components/layout";
 
 
 const useExpenseStore = create((set) => ({
```
Comment (thelmaboamah) [inline src/pages/expense.jsx]: I made a change to my goals store in this [PR](https://github.com/WorthyFinancial/WebApp/pull/1/files#diff-1a7ea470e4c1227305e6b25160922cb48f13666a158df94979a6ee1f909ae559R23). I'm learning Zustand on the fly but apparently wrapping your store with the useStore function zustand provides it the proper way to export it, though i can't track down where i read this.

Comment (keishaharris) [reply src/pages/expense.jsx]: ahh, okay thanks for the insight. If you happen to find the source let me know. 
This also looks much more organized so I'll also do the same and make the other changes you pointed out as well. Thanks, Thelma.

**Review by thelmaboamah — APPROVED** [review]
Looks good to me!

**Pull request merged**